### ISSUE TITLE
Possible to use with Distributed?

### ISSUE THREAD

**Issue by thomvet** [YingboMa/SafeTestsets.jl#13]
I am trying to test a Server/Client application and would like to employ SafeTestsets.jl. However, it seems that the use of @everywhere and @spawnat within a @safetestset causes previously working tests to fail. Is this a scoping issue? (Macros evaluating into Main instead of the module generated by SafeTestsets.jl?

Any advice on how to circumvent this issue?

**Comment by ChrisRackauckas** [comment]
Yes Distributed wants to work in Main and this adds a module. We generally just don't mix it with distributed for that reason, and leave distributed tests as the last test set without a safetestset for that issue.

**Comment by thomvet** [comment]
Ok, so that combination is just currently not possible then and I'll have to actually bother writing a clean testset myself rather than relying on this package to rescue me :) 

Thanks for the rapid response.

**Comment by spawnat** [comment]
why am I mentioned in this thread?

-spawnat

**Comment by thomvet** [comment]
Sorry, that was unintentional. There's an @ spawnat macro in Julia....
